packages/opentelemetry-plugin-winston/src/utils.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TraceInformation, TRACE_PARAM_NAME, WinstonChunk } from './types';
+import * as types from '@opentelemetry/types';
+
+/**
+ * Get trace information from span
+ * @param span
+ */
+function getTraceInformation(span: types.Span): TraceInformation {
+  const traceInformation: TraceInformation = {
+    trace_id: span.context().traceId,
+    span_id: span.context().spanId,
+  };
+  return traceInformation;
+}
+
+/**
+ * Adds information to chunk about trace
+ * @param chunk
+ * @param span
+ */
+export function addTraceToChunk(
+  chunk: WinstonChunk,
+  span: types.Span
+): WinstonChunk {
+  const newChunk: WinstonChunk = Object.assign({}, chunk);
```

Comment:
Can we check if a property named TRACE_PARAM_NAME already exists and skip if it does? I don't want to modify the user's application behavior if they're already logging some property named `ot`.